src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -179,11 +179,13 @@ class FacetSidebar extends React.Component {
                 return (
                   <label
                     key={j}
+                    id={`${facet.field}-${valueLabel}`}
                     htmlFor={`${facet.field}-${valueLabel}`}
                     className={`nypl-bar_${percentage}`}
                   >
                     <input
                       id={`${facet.field}-${valueLabel}`}
+                      aria-labelledby={`${facet.field} ${valueLabel}`}
```

Comment:
Sorry, to clarify, our *current version* of React wraps it in a span. This is fixed in later versions of React, but we are running 14.8, where the problem is still present.